db/migrate/20190621160800_rename_host_group_matchers.rb

```diff
@@ -0,0 +1,23 @@
+class RenameHostGroupMatchers < ActiveRecord::Migration[5.2]
+  def up
+    host_group_matchers = Setting.find_by_name('host_group_matchers_inheritance')
+    matchers_inheritance = Setting.find_by_name('hostgroup_matchers_inheritance')
+    return unless host_group_matchers.present? && matchers_inheritance.present?
+    matchers_inheritance.update_attribute(
+      :value,
+      host_group_matchers.value
+    )
+    host_group_matchers.destroy
+  end
+
+  def down
+    host_group_matchers = Setting.find_by_name('host_group_matchers_inheritance')
+    matchers_inheritance = Setting.find_by_name('hostgroup_matchers_inheritance')
+    return unless host_group_matchers.present? && matchers_inheritance.present?
```

Comment:
When rolling back the `host_group_matchers_inheritance` setting should not exist, as it was destroyed by the up migration and will not be recreated automatically by initialization once this change is merged. So in the down migration it should be possible to rename the setting instead (but we can check to make sure the old name doesn't exist first and drop it if it does before renaming the new one to the old name)